proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -166,3 +205,21 @@ message SearchDraftResponse {
   // There are no more pages if this is empty.
   string next_page_token = 2;
 }
+
+message SearchContentRequest {
```

Comment:
Should this explicitly take the repo owner + name (or id), so we can filter down to content within the scope of that module?

Reply:
I thought about this (@pgmitche suggested this when we were discussing), and decided to add this later when we flushed out in our preferred UX and confirmed that there is a place for it, but this is definitely a reasonable ask

Reply:
I guess it depends where this search feature is going to live within the UI. I assume it'll be scoped to a given repository, but could be wrong.

For searching content I imagine there are at least 3 levels (if not 4):

- global (entire server)
- organization
- repository
- _maybe_ even per package

If this is indeed scoped to any/all of the above, we should strongly consider starting with accepting an input and filter down as much as possible to only the scope we need.

Reply:
discussed offline, adding these in the request